reboot.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Linux Reboot Demo
```

Comment:
Any way we can name this something like `Reboot k3s_cluster`, rather than demo?  Thank you!

Reply:
Yeah, That would be better!!! Just updated the name. Thank You :)

Reply:
Thank you so much!